Formula/rmtrash.rb

```diff
@@ -0,0 +1,17 @@
+class Rmtrash < Formula
+  desc "Command rmtrash moves directory entries to Trash on macOS"
+  homepage "https://changkun.de/s/rmtrash"
+  url "https://github.com/changkun/rmtrash/archive/v0.0.1.tar.gz"
+  sha256 "49f8034f334f46b1b5c9acbc177b43fe669036da7b1ce516faa8866aa57b9c39"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", "#{bin}/rmtrash"
+  end
+
+  test do
+    system "#{bin}/rmtrash", "-v"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.